cli/sql.go

```diff
@@ -49,22 +56,52 @@ func runTerm(cmd *cobra.Command, args []string) {
 		_ = liner.Close()
 	}()
 
+	fmt.Fprint(osStdout, infoMessage)
+
+	// TODO(marc): allow passing statements on the command line,
+	// or specifying a flag. This would make testing much simpler.
+	// TODO(marc): add test.
+	var stmt string
 	for {
-		// TODO(marc): handle multi-line, this will require ';' terminated statements.
-		l, err := liner.Prompt("> ")
+		prompt := "> "
+		if len(stmt) != 0 {
+			// We're continuing a statement: indent relative to beginning.
+			prompt = "    "
+		}
+
+		l, err := liner.Prompt(prompt)
 		if err != nil {
 			if err != io.EOF {
-				fmt.Fprintf(os.Stderr, "Input error: %s\n", err)
+				fmt.Fprintf(osStderr, "Input error: %s\n", err)
 			}
 			break
 		}
-		if len(l) == 0 {
+
+		// See if we have a semicolon at the end of the line (ignoring
+		// whitespace after it).
+		tmpLine := strings.TrimSpace(l)
```

Comment:
Move this after `stmt += " " + l` to keep all uses of `tmpLine` together. (blank lines will result in extra spaces, but they're harmless).

You can get rid of the variable completely: `if strings.HasSuffix(strings.TrimSpace(l), ";")`